cmd/kube-proxy/app/server.go

```diff
@@ -46,13 +46,16 @@ import (
 	"k8s.io/client-go/tools/record"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/apis/componentconfig"
-	"k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1"
 	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
 	informers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/kubelet/qos"
 	"k8s.io/kubernetes/pkg/master/ports"
 	"k8s.io/kubernetes/pkg/proxy"
+	proxycomponentconfig "k8s.io/kubernetes/pkg/proxy/apis/proxyconfig"
```

Comment:
You don't have to change this, but I would probably call this `proxyconfigapi`. Or just import it without aliasing, and unalias the `proxyconfig` below.

Reply:
Done. (all associated importing without aliasing in server*.go) Thanks!